dark/data/gor4/README.md

```diff
@@ -0,0 +1,8 @@
+## Files in this directory
+
+`New_KS.267.obs` and `New_KS.267.seq` are the original database files that
+come with the GOR4 code.
+
+`new-gor4.obs` and `new-gor4.seq` are new database files for GOR4 which were
+made according to the criteria outlined in [GOR Method for Predicting Protein Secondary Structure from Amino Acid Sequence](http://www.ulb.ac.be/di/map/tlenaert/Home_Tom_Lenaerts/INFO-F-208_files/1996%20Garnier.pdf).
+For a detailed description, see [Updating the GOR4 databases, 28/6/2016](https://notebooks.antigenic-cartography.org/barbara/pages/features/updating-gor4-databases.html).
```

Comment:
I think it would be good to put the description into this file. The URL you give will not be accessible to the outside world. Assuming it's not too complicated to do that.